python/test/test_query.py

```diff
@@ -0,0 +1,61 @@
+import json
+import time
+
+from timeplus import Query
+
+
+def test_query_with_stream(test_environment, test_stream):
+    # Test the Query object with stream
+    query = (
+        Query(env=test_environment)
+        .sql(query=f"SELECT * FROM test_stream")
+        .create()
+    )
+
+    query_id = query.id()
+
+    query_list = Query(env=test_environment).list()
+    assert query_id in [q.id for q in query_list]
+
+    metadata = query.metadata()
+    assert "name" in metadata
+    assert "sql" in metadata
+    assert "id" in metadata
+
+    query.cancel()
+    # Without query.cancel(), there will be an error:
+    # OSError: [Errno 9] Bad file descriptor And the query cannot be deleted
```

Comment:
just create a issue with https://github.com/timeplus-io/gluon/issues and add the issue link